### PR TITLE
[JBPM-9920] Trying to avoid call to dispose in tests

### PR DESCRIPTION
**Pull request by fjtirado**
This is a draft so testing folks can try if they need to call dispose with that change. Once this is verified, we will see how to proceed "officially"

### CODE REVIEW

File: jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/AbstractRuntimeManager.java [hunk 1]
```diff
@@ -178,13 +177,9 @@ protected void initTimerService() {
                 }
             }
         }
-        scheduleDeadlines();
```
[the remark this answers is not in the document]
Comment (fjtirado) [reply jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/AbstractRuntimeManager.java:181]: I agree with initTimer, but I was thinking that maybe Init Deadlines should be done per deployment, at the end of common deploy. This way it does not matter which strategy we use for runtime, when a kie jar is deployed, the deadlines associated to process in that deployment are then scheduled.

**Review by afalhambra — APPROVED** [review]
Looks good to me, integration tests are working fine now. 
Nice work @fjtirado

File: jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/AbstractRuntimeManager.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (elguardian) [reply jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/AbstractRuntimeManager.java:181]: Keep in mind that the logicical place for this is the Runtime manager... if you move this to the services it means that the initilization of the timers would only be possible through jbpm service layer which is not correct as this is part of the core engine and not the service layer.

**Comment by mbiarnes** [comment]
Jenkins retest this

**Comment by mbiarnes** [comment]
Jenkins retest this

**Comment by afalhambra** [comment]
Jenkins retest this

**Comment by fjtirado** [comment]
jenkins retest this

**Comment by fjtirado** [comment]
jenkins retest this

**Pull request merged**